.ci/Jenkinsfile

```
@@ -142,7 +156,10 @@ pipeline {
             def parallelTasks = [:]
             agents['agents'].each { agent ->
               if (agent.JSON_SPECS_PATH) {
-                parallelTasks["${agent.REPO}"] = generateStepForAgent(repo: "${agent.REPO}", filesType: "json", filesPath: "${agent.JSON_SPECS_PATH}")
+                def repository = 'apm-agent-' + agent.REPO
+                if (env.SELECTED_AGENT == 'All' || env.SELECTED_AGENT == repository) {
```

See previous comment

```suggestion
                if (env.SELECTED_AGENT == 'all' || env.SELECTED_AGENT == agent.REPO) 
```